t/test_ocean_grid_gen.py

```diff
@@ -1,65 +1,111 @@
-#The checksums in this file were generated on gfdl PAN platform with 
-#commit ca730b83714630a05c4c8f133b23152a6165909a
+# The checksums in this file were generated on gfdl PAN platform with
```

Comment:
These testing tools under t/ are not meant to be read/used/modified by the users.

Reply:
just fixed a style detail, nothing more